tests/static_ci_tests.lst

```diff
@@ -0,0 +1,40 @@
+tests/DCPS/C++11/Messenger/run_test.pl: !DCPS_MIN CXX11 !STATIC_COMPILER !STATIC_UNIT
```

Comment:
Is there any reason these aren't specified as single-flag positives (e.g. "STATIC_MESSENGER") rather than multi-flag negatives (e.g. "!STATIC_COMPILER !STATIC_UNIT")?